man/man7/fi_cxi.7

```diff
@@ -0,0 +1,2144 @@
+.\"t
```

Comment:
I believe this file should get generated automatically when upstream.

Reply:
The build failed when this wasn't present. I included it, but I can remove it.

Reply:
Maybe there is a chicken and egg issue here. Possibly the bot to create the man page output needs to be kicked off first this initial time.